TemplateApplicationUITests/SchedulerTests.swift

```diff
@@ -61,6 +61,13 @@ class SchedulerTests: XCTestCase {
         XCTAssert(app.staticTexts["E-mail"].exists)
         app.staticTexts["E-mail"].tap()
 
+        XCTAssert(nextButton.isEnabled)
+        nextButton.tap()
+        
+        // staticTexts["What is your e-mail?"] is not exposed in the accessibility hierarchy; it seems like a bug in ResearchKit.
```

Comment:
@vishnuravi This is kind of interesting; seems like RK is just completely missing the accessibility hints for these views, making them unaccessible to UI tests. But only appears for steps that are rendered based on conditional elements. We might be able to replicate this in RKoF; wondering if this is bug in our mapping or a RK bug itself.